### DISCUSSION TITLE
Keyboard backlight color when shutdown

### DISCUSSION THREAD

**Discussion by jacky3605** [seerge/g-helper#756] [Q&A]
Hello, my laptop model is ASUS TUF Gaming A15 FA507XV.
My problem is how can I make keyboard backlight color equally when running and shutdown?
Its original color is blue, but when shutdown it become white color...
I remember that it should be the same color when lights dimming out.
[https://imgur.com/a/WISThqg](url)

One day my laptop had iGPU driver problem,
I couldn't shutdown it, my screen became black,
I tried to press power button, but didn't work,
so  I pressed Shift + Power Button, it finally shutdown.
I think my keyboard backlight problem was start from here.
It might cause some keyboard setting reset.
I used Armoury Crate also could't fix this problem, too

I wish you can help me fix it, thank you!

**Comment by seerge** [comment]
@jacky3605 hello, don't think that boot / shutdown colors can be customized anyhow. They can be only turned on or off (via Extra screen).

**Reply by jacky3605** [reply]
OK, it should be the same color before when choosing color,
but now it won't change.

**Reply by jacky3605** [reply]
Maybe I should try to reinstall the driver?
Can you tell me which driver would related?

**Reply by seerge** [reply]
It's controlled by BIOS, not by a driver :) (as it happens w/o windows even )

**Reply by seerge** [reply]
You can try https://github.com/seerge/g-helper#how-do-i-do-a-hardware-reset-on-a-laptop

**Reply by jacky3605** [reply]
Unfortunately, it didn't work:(

**Reply by seerge** [reply]
Well, then i'm out of ideas:) you may check OpenRGB app to see if it offers any extra controls (most probably not)

**Reply by jacky3605** [reply]
Didn't work, too...
Thanks for your help anyway!
